Libs/MRML/CLI/vtkMRMLCommandLineModuleNode.cxx

```diff
@@ -1046,25 +1052,38 @@ void vtkMRMLCommandLineModuleNode::Modified()
 }
 
 //----------------------------------------------------------------------------
-const std::string vtkMRMLCommandLineModuleNode::GetErrorText() const
+std::string vtkMRMLCommandLineModuleNode::GetErrorText() const
 {
-  return this->Internal->ErrorText;
+  std::string text;
+    {
+    std::lock_guard<std::recursive_mutex> lk(this->Internal->NodeAccessMutex);
```

Comment:
Since I found reference to both in examples, I suggest to be explicit.

* Example with `lock`: https://en.cppreference.com/w/cpp/thread/lock_guard
* Example with `lk`: https://en.cppreference.com/w/cpp/thread/scoped_lock